### ISSUE TITLE
Starting Synthesis of "The 6"

### ISSUE THREAD

**Issue by azsherman** [OpenSourceMalaria/OSM_To_Do_List#428]
Olivia and I (Alyssa) will be working on the synthesis of "the 6" for our Superlab project at Haverford College this fall.  We will be following the synthetic scheme outlined in #401, pictured below.  
![screen shot 2016-09-02 at 9 18 28 am](https://cloud.githubusercontent.com/assets/21690014/18205115/544834d8-70ee-11e6-94fe-2836f7f75c1f.png)

**Comment by mattodd** [comment]
Am assuming your lab notebook is [here](http://malaria.ourexperiment.org/synthesis_of__the_6), @azsherman and Alyssa?

**Comment by mattodd** [comment]
Hi @azsherman - any updates to this synthesis? You were [trying out mono-methylations](http://malaria.ourexperiment.org/synthesis_of__the_6) a month or so ago. Any luck?

**Comment by azsherman** [comment]
Hi @mattodd. Unfortunately we have been having some issues with the methylation.  Originally we had been attempting to methylate using the procedure described in Scales et al. Organic Letters, 15(9), 2156-2159; 2013.  We attempted this reaction three separate times.  The first time the reaction appeared to work but did not give us enough product to move forward so we repeated the reaction but were unable to reproduce the results.  Instead of reacting, a white powder would crash out of the mixture which GC/MS showed to be our starting material.  We decided to turn to a different procedure where rather than methylating we start with a methylpyrazine and chlorinate it.  We are hoping to have results from this attempt by the end of the week.

**Comment by mattodd** [comment]
Sure, I see that, but I just wanted to try to make sure that these attempts, even though they didn't give you what you wanted, are detailed in the ELN, so that future people approaching this target can benefit from your experience.

**Comment by olivia-franco** [comment]
Alyssa and I have started an attempt at a different route using a different starting material. This reaction is two steps, we have done the first step and our spectra suggests that we have synthesized the desired intermediate. We will proceed with the second step Thursday or Friday this week. 

Reaction scheme:
<img width="639" alt="screen shot 2016-10-19 at 7 08 01 pm" src="https://cloud.githubusercontent.com/assets/20305297/19540726/6ecc4608-962f-11e6-9152-07e61f1c6d10.png">

**Comment by azsherman** [comment]
Yes sorry, we are a little behind on our ELN but we are working on updating it to reflect all of our trials and errors

**Comment by mattodd** [comment]
Am I right in thinking that this part of the project has been parked now, since there are no more lab book updates? If so, you can close this Issue, since all the links have been installed on the [DCNYS](http://openwetware.org/wiki/OpenSourceMalaria:Triazolopyrazine_%28TP%29_Series#Desirable_Compounds_Not_Yet_Synthesised) section of the wiki. It'd be useful to have a short summary somewhere of where you got up to, and any advice to students coming after you who might be interested in this target.
